sunpy/timeseries/sources/fermi_gbm.py

```diff
@@ -157,24 +159,24 @@ def _parse_hdus(cls, hdulist):
         count_data = hdulist[2].data
 
         # rebin the 128 energy channels into some summary ranges
+        # some of default bin ranges are
         # 4-15 keV, 15 - 25 keV, 25-50 keV, 50-100 keV, 100-300 keV, 300-800 keV, 800 - 2000 keV
         # put the data in the units of counts/s/keV
-        summary_counts = _bin_data_for_summary(energy_bins, count_data)
+        summary_counts = _bin_data_for_summary(energy_bins, count_data, **kwargs)
 
         # get the time information in datetime format with the correct MET adjustment
         met_ref_time = parse_time('2001-01-01 00:00')  # Mission elapsed time
         gbm_times = met_ref_time + TimeDelta(count_data['time'], format='sec')
         gbm_times.precision = 9
         gbm_times = gbm_times.isot.astype('datetime64')
 
-        column_labels = ['4-15 keV', '15-25 keV', '25-50 keV', '50-100 keV',
-                         '100-300 keV', '300-800 keV', '800-2000 keV']
+        ebands = kwargs.get('energy_bands', [4, 15, 25, 50, 100, 300, 800, 2000])
```

Comment:
The energy bins should have astropy units attached to them and we would want to enforce that.